crates/bvs-rewards/src/msg.rs

```diff
@@ -0,0 +1,84 @@
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use cosmwasm_std::{HexBinary, Uint128};
+
+#[cw_serde]
+pub struct InstantiateMsg {
+    /// Owner of this contract
+    pub owner: String,
+}
+
+#[cw_serde]
+pub enum ExecuteMsg {
+    DistributeRewards {
+        merkle_root: HexBinary,
+        reward_distribution: RewardDistribution,
+        reward_type: RewardsType,
+    },
+    ClaimRewards {
+        claim_rewards_proof: ClaimRewardsProof,
+        reward_type: RewardsType,
+        service: String,
+        /// token refers to the address of the token contract (CW20) or denom string (Bank)
+        token: String,
+        /// amount refers to the total amount of rewards accrued to the user
+        amount: Uint128,
+        recipient: String,
+    },
+    TransferOwnership {
+        /// See [`bvs_library::ownership::transfer_ownership`] for more information on this field
+        new_owner: String,
+    },
+}
+
+#[cw_serde]
+#[derive(QueryResponses)]
+pub enum QueryMsg {
+    #[returns(DistributionRootResponse)]
+    DistributionRoot { service: String, token: String },
+    #[returns(BalanceResponse)]
+    Balance { service: String, token: String },
+    #[returns(ClaimRewardsResponse)]
+    ClaimedRewards {
+        service: String,
+        token: String,
+        earner: String,
+    },
+}
+
+#[cw_serde]
+pub struct DistributionRootResponse(pub String);
+
+#[cw_serde]
+pub struct BalanceResponse(pub Uint128);
+
+#[cw_serde]
+pub struct ClaimRewardsResponse(pub Uint128);
+
+#[cw_serde]
+pub enum RewardsType {
+    CW20,
+    Bank,
```

Comment:
```suggestion
    Cw20,
    Bank,
```

Due to rust initialism rules, this must be Cw20 instead of CW20.